dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/IndiciesInfo.java

```diff
@@ -0,0 +1,234 @@
+package com.dotcms.content.elasticsearch.business;
+
+import com.dotcms.enterprise.cluster.ClusterFactory;
+import com.dotmarketing.exception.DotRuntimeException;
+import io.vavr.control.Try;
+
+import java.io.Serializable;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.commons.beanutils.PropertyUtils;
+
+/**
+ * Handle the current Index name into the cluster.
+ * Also provide some util method for index name, the index name has to have the follow sintax:
+ *
+ * <b>cluster_<CLUSTER_ID>.<INDEX_TYPE_PREFIX>_<TIME_STAMP></b>
+ *
+ * where:
+ *
+ * - CLUSTER_ID: it is the cluster identifier who created the index.
+ * - INDEX_TYPE_PREFIX: Prefix define for the Index's typs, also see {@link IndexType}
+ * - TIME_STAMP: current time stamop when the index was created
+ */
+public class IndiciesInfo implements Serializable {
+
+    /**
+     * Build a new {@link IndiciesInfo} instance
+     */
+    public static class Builder {
+        private String live, working, reindexLive, reindexWorking, siteSearch;
+
+        public Builder setLive(String live) {
+            this.live = live;
+            return this;
+        }
+
+        public Builder setWorking(String working) {
+            this.working = working;
+            return this;
+        }
+
+        public Builder setReindexLive(String reindexLive) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'reindexLive' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)